package.json

```diff
@@ -22,11 +22,17 @@
     "isomorphic"
   ],
   "author": "Reza Akhavan <jedireza@gmail.com> (http://reza.akhavan.me/)",
+  "contributors": [
+    "Tane Piper <piper.tane@gmail.com> (https://github.com/tanepiper)"
+  ],
```

Comment:
We can remove `contributors` from the package. We have something much better here: https://github.com/jedireza/hapi-react-views/graphs/contributors